test/jdk/javax/swing/JTabbedPane/bug4666224.java

```diff
@@ -49,7 +49,7 @@ public class bug4666224 {
     private static JFrame frame;
 
     private static final String INSTRUCTIONS = """
-                ON ALL PLATFORMS
+                ON ALL PLATFORMS except macos where pt.6 is not applicable
```

Comment:
Instead of mentioning about the exception, I guess you can move it to point 10 and exclude/include from test instructions (Basically string concat) based on OS type?

Reply:
No, that is in logical sequence of testing steps...we probably can do pt1-5+conditional pt 6+pt7-10 but I think this should be fine as it is mentioned upfront...

Reply:
Yeah, agree that moving it to last makes it little out of sequence while testing. Do you have any reference of native application in windows where resizing wraps tabbed pane? I checked in command prompt of windows, it doesn't resize either?

Reply:
No, I dont know of any native apps in windows but probably you can verify in Settings->Home where resizing window wraps the tabs..

Reply:
Sure. Just wanted to make sure its not product bug, since we are verifying w.r.t native applications.

Reply:
That macos doesn't support tab policy namely SCROLL_TAB and WRAP_TAB can be further seen in this [JDK-8263916](https://bugs.openjdk.org/browse/JDK-8263916) statement "As macOS does not have the concept of a tab view with scrolling tabs," which in this case also refers to wrapping tabs..

Reply:
Well, in that case it looks good to me.